resources/testharness.js

```diff
@@ -520,6 +520,8 @@ policies and contribution forms [3].
             Object.prototype.toString.call(worker) == '[object ServiceWorker]';
     }
 
+    var seen_func_name = Object.create(null);
```

Comment:
I might have just used `new Map`

Reply:
Given in general we've been pretty conservative about browser support for testharness.js I decided to do what we did elsewhere. 🙃